internal/server/key.go

```diff
@@ -55,7 +55,7 @@ func buildStatsKey(
 	return rowList, keyToToken
 }
 
-func buildStatCollectionKey(parentPlace, childType, date string, statVars []string) (
+func buildStatSetWithinPlaceKey(parentPlace, childType, date string, statVars []string) (
```

Comment:
Is there any caller with date set?  If not, should we then stop building this ObsCollection cache?

Reply:
"d/f/" cache is not used now (both w/o date). We can disable the cache build for now (maybe keep the code?).